verso/src/lib.rs

```diff
@@ -3,23 +3,49 @@ use versoview_messages::ControllerMessage;
 
 use ipc_channel::ipc::{IpcOneShotServer, IpcSender};
 
+#[derive(Debug, Default)]
+pub struct VersoviewSettings {
+    pub with_panel: bool,
+}
+
 pub struct VersoviewController(IpcSender<ControllerMessage>);
 
 impl VersoviewController {
-    /// Create a new verso instance and get the controller to it
-    pub fn new(verso_path: impl AsRef<Path>, initial_url: url::Url) -> Self {
+    /// Create a new verso instance with settings and get the controller to it
+    fn create(
```

Comment:
I think we can leave this as `new`. If developers don't want to modify the settings, then they can just pass `VersoViewSettings::default()`

Reply:
I kinda want this for the simplicity so anyone who wants to use it could just pass in the required fields without having to care about the settings